### PR TITLE
Make SendChatActionParams.action an enum

### LINKED ISSUE #21: Make SendChatActionParams.action an enum
* `::Typing` for text messages
* `::UploadPhoto` for photos
* `::RecordVideo` or `::UploadVideo` for videos
* `::RecordVoice` or `::UploadVoice` for voice notes
* `::UploadDocument` for general files
* `::FindLocation` for location data
* `::RecordVideoNote` or `UploadVideoNote` for video note

https://core.telegram.org/bots/api#sendchataction

### PR DESCRIPTION
**Pull request by Defman21**
Fixes #21

### CODE REVIEW

File: src/api_params.rs [hunk 1]
```diff
@@ -122,6 +122,21 @@ pub enum ReplyMarkup {
     ForceReply(ForceReply),
 }
 
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq)]
+#[serde(rename_all = "snake_case")]
+pub enum ChatAction {
+    Typing,
+    UploadPhoto,
+    RecordVide,
```
Comment (ayrat555) [inline src/api_params.rs:130]: typo

**Review by ayrat555 — APPROVED** [review]
Thank you for your contribution.

File: src/api_params.rs (hunk 1 above)
Comment (Defman21) [reply src/api_params.rs:130]: Oops, I've fixed it locally but forgot to commit.

**Comment by ayrat555** [comment]
A new version (0.5.2) was released

**Pull request merged**